Loenn/lang/en_gb.lang

```diff
@@ -153,3 +153,35 @@ entities.FactoryHelper/RustyLamp.attributes.description.activationId=String valu
+triggers.FactoryHelper/FactoryActivationTrigger.placements.name.factory_activation=Factory Activation
+triggers.FactoryHelper/FactoryActivationTrigger.attributes.description.activationIds=A comma separated set of string Ids for all the Activation Ids this trigger should activate.
+triggers.FactoryHelper/FactoryActivationTrigger.attributes.description.ownActivationId=String value of the trigger's Activation Id. Can only fire if the Id is active.
+triggers.FactoryHelper/FactoryActivationTrigger.attributes.description.resetOnLeave=Determines whether the Ids activated should be reset to their previous state upon leaving. If Persistent is true this will be ignored.
+triggers.FactoryHelper/FactoryActivationTrigger.attributes.description.persistent=Determines whether the Ids not already persistent that are governed by this trigger should become persistent, meaning they will be active even after leaving the room.
+
+# Factory Event Trigger
+triggers.FactoryHelper/EventTrigger.placements.name.factory_event=Factory Event
+triggers.FactoryHelper/EventTrigger.attributes.description.event=String Id of the event to be triggered.
+
+# Permanent Activation Trigger
+triggers.FactoryHelper/PermanentActivationTrigger.placements.name.permanent_activation=Permanent Activation
+triggers.FactoryHelper/PermanentActivationTrigger.attributes.description.activationIds=A comma separated set of string Ids for all the Activation Ids this trigger should activate.
+
+# Spawn Steam Wall Trigger
+triggers.FactoryHelper/SpawnSteamWallTrigger.placements.name.spawn_steam_wall=Spawn Steam Wall
+triggers.FactoryHelper/SpawnSteamWallTrigger.attributes.description.color=Hex color code (example: ff0000 is red) of the spawned steam wall. Default is white.
+triggers.FactoryHelper/SpawnSteamWallTrigger.attributes.description.speed=Speed multiplier for the spawned wall's movement. Default speed is 22 pixels per second. A tile is 8 pixels wide.
+
+# Special Box Deactivation Trigger
+triggers.FactoryHelper/SpecialBoxDeactivationTrigger.placements.name.special_box_deactivation=Special Box Deactivation
+
+# Steam Wall Color Trigger
+triggers.FactoryHelper/SteamWallColorTrigger.placements.name.steam_wall_color=Steam Wall Color
+triggers.FactoryHelper/SteamWallColorTrigger.attributes.description.color=Hex color code (example: ff0000 is red) to change the color of the spawned steam wall. Has no effect if there is no steam wall currently spawned. Default is white.
+triggers.FactoryHelper/SteamWallColorTrigger.attributes.description.duration=The amount of time it will take for the color change to occur in seconds.
+
+# Steam Wall Speed Trigger
+triggers.FactoryHelper/SteamWallSpeedTrigger.placements.name.steam_wall_speed=Steam Wall Speed
+triggers.FactoryHelper/SteamWallSpeedTrigger.attributes.description.speed=Changes the speed multiplier a spawned wall movement. Has no effect if there is no steam wall currently spawned. Default speed is 22 pixels per second. A tile is 8 pixels wide.
```

Comment:
`Changes the speed multiplier a spawned wall movement` ?

also I would set the attribute name to "Speed Multiplier" or something so it's clear you're not setting it directly (also applies to the spawn trigger)

you can do that w/o changing the field itself by setting `attributes.names.speed`